corehq/apps/cloudcare/templates/formplayer/dependencies.html

```diff
@@ -35,6 +35,32 @@
   <script src="{% static 'clipboard/dist/clipboard.js' %}"></script>
   <script src="{% static 'reports/js/readable_form.js' %}"></script>
   <script src="{% static 'url-polyfill/url-polyfill.js' %}"></script>
+  <script>
+      function getCookie(cname) {
+        let name = cname + "=";
+        let decodedCookie = decodeURIComponent(document.cookie);
+        let ca = decodedCookie.split(';');
+        for(let i = 0; i <ca.length; i++) {
+          let c = ca[i];
+          while (c.charAt(0) == ' ') {
+            c = c.substring(1);
+          }
+          if (c.indexOf(name) == 0) {
+            return c.substring(name.length, c.length);
+          }
+        }
+        return "";
+      }
+      $.ajaxSetup({
+        beforeSend: function(xhr, settings) {
```

Comment:
Could this be combined with the CSRF handling in `hq.helpers`? It'd be nice to have it in one place.

https://github.com/dimagi/commcare-hq/blob/7b0f35c49d2b9ae7cf60d6b41c9d4db3a7c45e1a/corehq/apps/hqwebapp/static/hqwebapp/js/hq.helpers.js#L148

Reply:
awesome, thanks for linking that @orangejenny. Definitely makes sense to have all the code in one place.